src/test/kotlin/io/snyk/plugin/services/SnykCliServiceTest.kt

```diff
@@ -391,6 +392,60 @@ class SnykCliServiceTest : LightPlatformTestCase() {
         assertEquals("", rawErrorCliResult.error!!.path)
     }
 
+    @Test
+    fun testConvertRawCliStringWithLicenseVulnsToCliResult() {
+        val cli = getCli(project)
+
+        val rawMissedFixedInFieldCliString = getResourceAsString("licence-vulnerabilities.json")
+        val cliResult = cli.convertRawCliStringToCliResult(rawMissedFixedInFieldCliString, "")
+        assertTrue(cliResult.isSuccessful())
+        assertNotNull(cliResult.allCliVulnerabilities?.find { it.vulnerabilities.any { it.fixedIn == null } })
+
+        touchAllFields(cliResult)
```

Comment:
@ArtsiomCh, what the method `touchAllFields` does?

Reply:
@pavel-snyk touchAllFields imitate UI reading fields process, i.e. we "touch" all fields to make sure it available and not producing any exception (like late initialization Exception).

Reply:
@ArtsiomCh,
as best practice unit test should test production code and not methods inside test classes, but I see your intention, so would say it's ok in this case.